code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -22,16 +22,42 @@
 		else
 			to_chat(user, "<span class='notice'>All [R]'s systems are nominal.</span>")
 
-	if(istype(M,/mob/living/carbon/human))		//Repairing robolimbs
+	if(istype(M,/mob/living/carbon/human)) //Repairing robotic limbs and IPCs
 		var/mob/living/carbon/human/H = M
 		var/obj/item/organ/external/S = H.get_organ(user.zone_sel.selecting)
 
 		if(S && S.is_robotic())
 			if(S.get_damage())
-				S.heal_damage(15, 15, robo_repair = 1)
 				use(1)
-				user.visible_message("<span class='notice'>\The [user] applies some nanite paste at[user != M ? " \the [M]'s" : " \the"][S.name] with \the [src].</span>",\
-				"<span class='notice'>You apply some nanite paste at [user == M ? "your" : "[M]'s"] [S.name].</span>")
+				var/remheal = 15
```

Comment:
Something about putting this here seems off but i am not sure if its technically snowflake.